Cargo.lock

```diff
@@ -2,6 +2,12 @@
 # It is not intended for manual editing.
```

Comment:
I've found conflicting thoughts on this online: should we be ignoring `cargo.lock`?

Reply:
Very large diff here and not sure if necessary

Reply:
It doesnt particularly matter its just our dependencies. We keep our yarn.lock and package-lock.json in our other projects so we might as well keep it here. Ideally if we dont chnage our dependencies this wont change either.